### ISSUE TITLE
Cannot delete old contacts

### ISSUE THREAD

**Issue by Norbert80** [deltachat/deltachat-desktop#1578]
Hi,

there seems to be no way to delete these contact items.

DeltaChat Desktop Version 1.2.0 on Ubuntu.

![grafik](https://user-images.githubusercontent.com/6921696/80287302-1beebf00-8731-11ea-814e-2c67399a2102.png)

**Comment by Jikstra** [comment]
Yup that's currently not possible. A conctact view to alter the contacts would be a good thing to have probably. I'm currently unsure if it's even possible on Android?

What you of course can do is block those contacts. Then they won't be shown in the New Chat dialog. You can click on the contact, then it opens the chat view with them and in the three dot menu in the upper left click on "Kontakt blockieren"/"Block contact". To unblock them you can go in the same menu on the "Blockierte Kontakte" entry.

**Comment by Jikstra** [comment]
Will close this issue as it's mainly a feature request and not a bug :)

**Comment by r10s** [comment]
>  I'm currently unsure if it's even possible on Android?

you can delete contacts on android/ios [as long as they are not referenced](https://github.com/deltachat/deltachat-core/issues/687).

> Will close this issue as it's mainly a feature request and not a bug :)

i agree, also, as the list is more a suggestion-list than an address-book, this feature is not that important, however, of course, nice to have one day :)
